### PR TITLE
[Doc] Describe clean consumed feature

### PR DESCRIPTION
**Pull request by andsel**
Describe the `clean_consumed` feature introduced with #43 and #45

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -57,6 +58,16 @@ input plugins.
...
 
+[id="plugins-{type}s-{plugin}-clean_consumed"]
+===== `clean_consumed`
+
+  * Value type is <<boolean,boolean>>
+  * Default value is `false`
+
+Deletes the DLQ segments that has been read, when it's configured.
+This feature implicitly requires `commit_offsets` set to `true`, if it's not then result in a configuration error.
+It's supported only for Logstash >= 8.4.0. If this setting is `true` and Logstash version doesn't provides this feature then result in a configuration error.
```
Comment (karenzone) [inline docs/index.asciidoc]:
```suggestion
When set to `true`, this option deletes the DLQ segments that have been read.
This feature requires that `commit_offsets` is set to `true`. If not, you'll get a configuration error.
This feature is available in Logstash 8.4.0 and later. If this setting is `true` and and you are using a Logstash version older than 8.4.0, then you'll get a configuration error.
```

File: lib/logstash/inputs/dead_letter_queue.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@ class LogStash::Inputs::DeadLetterQueue < LogStash::Inputs::Base
   # If true deletes the DLQ segments that has been processed.
   # Supported only Logstash >= 8.4.0
   # If this setting is `true` and Logstash version doesn't provides this feature then result in a configuration error.
-  # When set to `true` implicitly forces `commit_offsets` to `true`.
+  # This feature implicitly requires `commit_offsets` set to `true`, if it's not then result in a configuration error.
```
Comment (karenzone) [inline lib/logstash/inputs/dead_letter_queue.rb]:
```suggestion
  # This feature implicitly requires that the `commit_offsets` option is set to `true`. Iif it's not then you'll get a configuration error.
```

**Review by karenzone — COMMENTED** [review]
Minor wording suggestions inline for consideration.

**Comment by roaksoax** [comment]
Jenkins test this again

**Pull request merged**